src/JsendExceptionFormatter.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Shalvah\LaravelJsend;
+
+use Exception;
+use Illuminate\Validation\ValidationException;
+
+/**
+ * @mixin \App\Exceptions\Handler
+ */
+trait JsendExceptionFormatter
+{
+    /**
+     * Convert a validation exception into a JSON response.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Illuminate\Validation\ValidationException  $exception
+     * @return \Illuminate\Http\JsonResponse
+     */
+    protected function invalidJson($request, ValidationException $exception)
+    {
+        return jsend_fail(
+            $exception->errors(),
+            $exception->status
+        );
+    }
+
+    /**
+     * Prepare a JSON response for the given exception.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Exception $e
+     * @return \Illuminate\Http\JsonResponse
+     */
+    protected function prepareJsonResponse($request, Exception $e)
+    {
+        return jsend_error(
+            $e->getMessage(),
+            $e->getCode(),
+            null,
```

Comment:
Wouldn't this be more useful if the data is:

```suggestion
           ['stacktrace' => $e->getTrace()],
```

Of course, we can hide the stacktrace if not in debug mode.

Reply:
@shalvah good point. I've updated the code, and also mirrored some additional behavior from Laravel core.

Reply:
Nicely done!👏